src/api/mockApiData.ts

```diff
@@ -16,116 +16,211 @@ const mockApi = {
+    //   components: [
+    //     {
+    //       id: 'asdf2-asdf1-ad143',
+    //       type: 'alerter',
+    //       flavor: 'slack',
+    //       name: 'slack_alerter',
+    //       config: {
+    //         slack_token: 'INFU1328UJ',
+    //         default_slack_channel_id: 184195,
+    //       },
+    //     },
+    //   ],
+    //   creator: 'Hamza Tahir',
+    //   creation_date: '2022-08-12T07:12:45.931Z',
+    // },
+    // {
+    //   name: 'string',
+    //   pipeline_config: {},
+    //   project_name: 'string',
+    //   id: 'string',
+    //   created_at: '2022-08-09T19:08:54.541Z',
+    //   organization_id: 'string',
+    //   user_id: 'string',
+    //   pipeline_runs: [
+    //     {
+    //       id: 'string',
+    //       user_id: 'string',
+    //       status: 'string',
+    //       start_time: '2022-08-09T19:08:54.541Z',
+    //       end_time: '2022-08-09T19:08:54.541Z',
+    //       workers: 0,
+    //       cpus_per_worker: 0,
+    //       datasource_commit_id: 'string',
+    //       pipeline_run_type: 'string',
+    //       pipeline_components: [
+    //         {
+    //           start_time: '2022-08-09T19:08:54.541Z',
+    //           end_time: '2022-08-09T19:08:54.541Z',
+    //           status: 'string',
+    //           component_type: 'string',
+    //         },
+    //       ],
+    //       processing_backend_id: 'string',
+    //       orchestration_backend_id: 'string',
+    //       training_backend_id: 'string',
+    //       serving_backend_id: 'string',
+    //       training_args: {},
+    //       serving_args: {},
+    //     },
+    //   ],
+    // },
   ],
 
   pipelineByIdMockResponse: {
-    name: 'string',
-    pipeline_config: {},
-    workspace_id: 'string',
-    id: 'string',
-    created_at: '2022-08-09T18:59:01.847Z',
+    id: '9lg2jk3',
     organization_id: 'string',
-    user_id: 'string',
-    pipeline_runs: [
+    name: 'prod_stack',
+    project_name: 'string',
+    components: [
       {
-        id: 'string',
-        user_id: 'string',
-        status: 'string',
-        start_time: '2022-08-09T18:59:01.847Z',
-        end_time: '2022-08-09T18:59:01.847Z',
-        workers: 0,
-        cpus_per_worker: 0,
-        datasource_commit_id: 'string',
-        pipeline_run_type: 'string',
-        pipeline_components: [
-          {
-            start_time: '2022-08-09T18:59:01.847Z',
-            end_time: '2022-08-09T18:59:01.847Z',
-            status: 'string',
-            component_type: 'string',
-          },
-        ],
-        processing_backend_id: 'string',
-        orchestration_backend_id: 'string',
-        training_backend_id: 'string',
-        serving_backend_id: 'string',
-        training_args: {},
-        serving_args: {},
+        id: 'asdf2-asdf1-ad143',
+        type: 'alerter',
+        flavor: 'slack',
+        name: 'slack_alerter',
+        config: {
+          slack_token: 'INFU1328UJ',
+          default_slack_channel_id: 184195,
+        },
```

Comment:
This is the old stack response structure, just want to confirm that we are not using it now?